### DISCUSSION TITLE
Feature Request: TTL 74173

### DISCUSSION THREAD

**Discussion by lshkim** [logisim-evolution/logisim-evolution#2057] [Ideas]
Could someone implement 74173 TTL chip?  Thank you!!!

**Comment by maehne** [comment]
@BunchOfBits: Could you maybe have a look at this request?

**Reply by BunchOfBits** [reply]
Done, see PR #2826

**Reply by lshkim** [reply]
Oh, thank you!

**Comment by BunchOfBits** [comment]
@maehne can you please close this discussion item since it has been implemented

**Comment by davidhutchens** [comment]
Closing because the request was completed with #2826.
